go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go v0.47.0 // indirect
+	cloud.google.com/go/storage v1.0.0
```

Comment:
:eyes:

Reply:
🤷‍♂ 

I didn't make this change on purpose, Go must have added it.

Reply:
this makes me feel a bit :( but we don't have to tackle it here XD

the joys of dependency tools XD